ctags/main/parse.c

```diff
@@ -91,9 +91,18 @@ extern void makeSimpleScopedTag (const vString* const name,
 
 extern parserDefinition* parserNew (const char* name)
 {
+	return parserNewFull (name, 0);
+}
+
+extern parserDefinition* parserNewFull (const char* name, char fileKind)
+{
 	parserDefinition* result = xCalloc (1, parserDefinition);
 	result->name = eStrdup (name);
+
+	/* TODO: implement custom file kind */
 	result->fileKind = &defaultFileKind;
+
+	result->enabled = TRUE;
```

Comment:
yeah I saw, but I'm wondering why it wasn't there before, and gets added here.  but if it's only because our "main" code doesn't check whether the parser is enabled enough, it's fair enough.